backend/src/test/java/gov/cdc/usds/simplereport/service/OrganizationServiceTest.java

```diff
@@ -167,6 +185,35 @@ void getOrganizationsAndFacility_filterByIdentityVerified_success() {
     assertTrue(unverifiedOrgIds.contains(unverifiedOrg.getExternalId()));
   }
 
+  @Test
+  @WithSimpleReportSiteAdminUser
+  void getFacilitiesIncludeArchived_includeArchived_success() {
+    Organization org = testDataFactory.createValidOrg();
+    Facility deletedFacility = testDataFactory.createArchivedFacility(org, "Delete me");
+    testDataFactory.createValidFacility(org, "Not deleted");
```

Comment:
I don't think you need this since you're not asserting on it. Or, maybe assert on this facility to ensure it's returned as well, or that the facilities size should be 2.

Reply:
Ok! Thank you for that suggestion! I'll assert that the size should be 1 since only the archived facility should be returned!